rasa/model_testing.py

```diff
@@ -196,6 +196,9 @@ async def test_nlu(
 
     rasa.shared.utils.io.create_directory(output_directory)
 
+    if not additional_arguments:
```

Comment:
can we please also add a test to avoid regression?

Reply:
While writing the test I realized something else. 

I'm pretty sure that the `model` and `nlu_data` parameters aren't optional as-is either. Can I remove the `Optional` type here?

Reply:
good catch 👍🏻 Yeah, but please check the usages of this function then